src/main/java/com/greghaskins/spectrum/internal/hooks/Hook.java

```diff
@@ -27,11 +27,40 @@ void accept(final Description description, final RunReporting<Description, Failu
       final Block block) throws Throwable;
 
   /**
+   * Override to return true if the inner block cannot report its own errors for some reason.
+   * @return true to suppress wrapping inner block in self-reporting
+   */
+  default boolean requiresUnreportedInnerBlock() {
+    return false;
+  }
```

Comment:
Hooks were being heavily wrapped in exception handling/reporting, but this is not necessary for all hooks.